js/languagePickerModel.js

```diff
@@ -9,10 +9,10 @@ define([
       "_isEnabled": false,
       "displayTitle": "",
       "body": "",
-      "_languages": [],
       "_defaultLanguage": "en",
       "_defaultDirection": "ltr",
```

Comment:
As @moloko mentioned in his earlier comment, please can we bin off `_defaultLanguage` and `_defaultDirection`; these attributes are no longer used.